app/views/policy_dashboard/_policy_reports.html.erb

```diff
@@ -11,7 +11,7 @@
       <th><%= _("Other") %></th>
       <th><%= _('Actions') %></th>
     </tr>
-    <% @policy.arf_reports.latest.each do |arf_report| %>
+    <% @latest_reports.each do |arf_report| %>
```

Comment:
doesn't this reintroduce the behavior changed by https://github.com/theforeman/foreman_openscap/pull/377 ?

Reply:
It does revert this particular line, but I cannot remember why it was changed in the first place. It should return the same results though as it calls `latest` scope for the reports belonging to a policy, I added a test. The only difference is `@policy.arf_reports.latest` gives me `undefined method `total_entries' for #<ForemanOpenscap::ArfReport::ActiveRecord_AssociationRelation:0x007efd8d475b58>` so I cannot paginate on that.